workshops/boost-research-reproducibility-binder/paper.md

```diff
@@ -0,0 +1,113 @@
+They include the workshop advertisement, pre-event communication emails, agenda, instructor guide, a folder of workshop presentations, a paired example exercise to explore example Binderised notebooks, a feedback form and report to help measure the impact of the event and how to improve in the future. 
+The presentations include an introduction to the workshop, slides on reproducible computational environments and a Zero to Binder code-along built heavily off the [Build a Binder](https://build-a-binder.github.io/) workshop resources. 
+There are also slides on using Binder for fully reproducible research in R as an alternative to the Python-focused code-along. 
+At the end of the workshop, participants are able to take their own content (in a R or Jupyter notebook, or scripts that can be run in the terminal) and prepare it so that it can be used by others on [myBinder.org](https://mybinder.org/). 
+Individuals unable to attend a workshop can also work through these resources to improve their understanding of reproducible computational environments and how to prepare their project for sharing with Binder. 
+This material is aimed at aimed at researchers and others who are who are interested in reproducibility, containers or continuous integration; already familiar with R Markdown or Jupyter notebooks; and looking to communicate their research more effectively.
+
+The feedback report summarises our experience of using these materials in teaching and learning situations. On average, participants report an increase  in their understanding and confidence in using Binder, capturing their computational environment and reproducing their research. Areas that participants identified could be improved include highlighting the difficulties using R with Binder and in particular the tidyverse package which takes a long time to build, that the demonstrations in the afternoon were slower and that some time was taken during the workshop to explain Git terminology,  Markdown and help people get set up.
+
+# Statement of Need
+
+Reproducible research is necessary to ensure that scientific work can be trusted. 
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. 
+The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. 
+Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*[The Turing Way](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)* is a project [funded](https://www.turing.ac.uk/news/alan-turing-institute-spearhead-new-cutting-edge-data-science-and-artificial-intelligence) as part of UKRI’s Strategic Priorities fund. 
```

Comment:
The official acknowledgment @KirstieJane has added to the zenodo handbook record is "This work was supported by The UKRI Strategic Priorities Fund under the EPSRC Grant EP/T001569/1, particularly the "Tools, Practices and Systems" theme within that grant, and by The Alan Turing Institute under the EPSRC grant EP/N510129/1."

Reply:
👍 I've added an acknowledgement section.